src/modules/flight_mode_manager/tasks/AutoFollowTarget/FlightTaskAutoFollowTarget.cpp

```diff
@@ -187,22 +189,16 @@ Vector3f FlightTaskAutoFollowTarget::calculate_drone_desired_position(Vector3f t
 	return drone_desired_position;
 }
 
-Vector3f FlightTaskAutoFollowTarget::calculate_target_position_filtered(Vector3f pos_ned_est, Vector3f vel_ned_est,
-		Vector3f acc_ned_est)
+Vector3f FlightTaskAutoFollowTarget::calculate_target_position_filtered(Vector3f pos_ned_est, Vector3f vel_ned_est)
```

Comment:
Similar comment on position vs pose - maybe the method could be called `update_filtered_target_pose()` ? Here note you probably don't need to return anything.. just `.getState()` and `.getRate()` each time further down in the code you need the filtered position or velocity. The vector is already stored once there, no need for a copy.

On another note.. if there is only the reset check happening in this method, would also be valid to move it out into the main `update()` function.